src/main/java/org/jenkinsci/plugins/matrixauth/integrations/casc/MatrixAuthorizationStrategyConfigurator.java

```diff
@@ -30,11 +33,16 @@
     @Override
     @Nonnull
     public Set<Attribute<T, ?>> describe() {
-        return Collections.singleton(
-                new MultivaluedAttribute<T, String>("grantedPermissions", String.class)
+        return new HashSet<>(Arrays.asList(
+                new MultivaluedAttribute<T, String>("permissions", String.class)
                         .getter(MatrixAuthorizationStrategyConfigurator::getGrantedPermissions)
-                        .setter(MatrixAuthorizationStrategyConfigurator::setGrantedPermissions)
-        );
+                        .setter(MatrixAuthorizationStrategyConfigurator::setGrantedPermissions),
```

Comment:
Should the get and set not be renamed too?

Reply:
Unsure. Ultimately they end up setting the items in the `grantedPermissions` field.